From b9cbfb524d73ca953604dc421098b4a3aa14d095 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 30 Jun 2021 09:38:46 +0900
Subject: [PATCH] ksmbd: change server state type macro to enumeration
Git-commit: b9cbfb524d73ca953604dc421098b4a3aa14d095
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Change server state type macro to enumeration.

Reviewed-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/ksmbd/server.h | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/fs/ksmbd/server.h b/fs/ksmbd/server.h
index b682d28963e8..2fce06e8b833 100644
--- a/fs/ksmbd/server.h
+++ b/fs/ksmbd/server.h
@@ -8,10 +8,15 @@
 
 #include "smbacl.h"
 
-#define SERVER_STATE_STARTING_UP	0
-#define SERVER_STATE_RUNNING		1
-#define SERVER_STATE_RESETTING		2
-#define SERVER_STATE_SHUTTING_DOWN	3
+/*
+ * Server state type
+ */
+enum {
+	SERVER_STATE_STARTING_UP,
+	SERVER_STATE_RUNNING,
+	SERVER_STATE_RESETTING,
+	SERVER_STATE_SHUTTING_DOWN,
+};
 
 #define SERVER_CONF_NETBIOS_NAME	0
 #define SERVER_CONF_SERVER_STRING	1
-- 
2.33.0

